src/registers/control.rs

```diff
@@ -3,6 +3,10 @@
 pub use super::model_specific::{Efer, EferFlags};
 use bitflags::bitflags;
 
+/// Various control flags modifying the basic operation of the CPU.
```

Comment:
The CR8 register is mostly about task priority, right? We should mention that in the docs.